### DISCUSSION TITLE
strange pop up behaviour

### DISCUSSION THREAD

**Discussion by snknts** [i3/i3#6168] [Q&A]
Apologies for the vagueness.

I'm using i3 with multiple monitors/workspaces 

When I run SQL Workbench.. the 'Select Connection Profile' pop-up window will occasionally (maybe even usually) open on a different workspace than its parent window.
(I'm running it via dmenu... 'bindsym $mod+r exec --no-startup-id dmenu_run') 

I can't seem to reliably reproduce it.. it doesn't always do it.. at the moment it seems to be alternating between the expected monitor (parent) and the left-most monitor.

Not sure where to go from here (or even what to search for)
Has anyone else seen similar behavior?

**Comment by danixland** [comment]
it happens to me as well with sublime_text, I have it run on my second monitor and the popup will appear on the primary monitor.. Also after I select a file in the popup, the main window disappears and I have to click on it from polybar to have it visible again

**Comment by calmarc** [comment]
And my pop-ups, gets re-openend high and higher .. i.e. Gimp etc. And for the size of the title-bar it gets shifted up each time you open the pop-up.
I3 'could' place it elsewhere (via config-rules), it does not matter, Gimp (etc) relocated them - but somehow makes a mistake with the height of the title-bar.

At least, it's a little bit annoying .. suddenly the pop-up's are out of the workspace. (of course you can still use keybindings to move them back.. but still).
